qiskit/algorithms/minimum_eigen_solvers/qaoa.py

```diff
@@ -141,6 +143,7 @@ def __init__(
             )
 
     def _check_operator_ansatz(self, operator: OperatorBase) -> OperatorBase:
+        # TODO: should return something?
```

Comment:
Also type annotation claims that it returns something but it doesn't

Reply:
Also here it shouldn't return anything 👍🏻